app/jinja_filters.py

```diff
@@ -133,7 +133,7 @@ def format_household_member_name_possessive(names):
 
 
 @blueprint.app_template_filter()
-def format_list(list_items):
+def format_list_for_html(list_items):
```

Comment:
Feel like a better name can be used for this to match the rest of the file?

Reply:
@MIslam92 what would you suggest

Reply:
format_underordered_list? similar to whats already being used i.e. format_multilined_string as its more explicit to what the function is actually doing